target/linux/lantiq/dts/FRITZ7312.dts

```diff
@@ -0,0 +1,140 @@
+/dts-v1/;
+
+#include "ar9.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "FRITZ7312 - 1&1 WLAN-MODEM";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+
+		led-internet = &info_green;
```

Comment:
please drop led-internet here. You do not have a distinct led to show the internet connection status. The use of a single led for dsl line status  and internet connection status is possible since 0481a2a2c5b85e82d3b7a55ee556c1a23dd1c135 and all affected boards were changed with 259fc1e77892ef8b7fb41c5bc118b55e31e0554b.